docs/pages.yml

```diff
@@ -21,3 +21,6 @@
     - Documenting Applications:
         - Creating Documentation: 'content/doc_generation/Overview.md'
         - Moose Flavored Markdown: 'content/doc_generation/MooseFlavoredMarkdown.md'
+- Wiki:
```

Comment:
I don't think I like the term "wiki" here, the reason being is that it is not informative to what it contains.

 I think we need a section called "tools" or "utilities" or something else where all of our non-MOOSE tools documentation site, including the documentation system itself. I would put this memory logger in that group.